<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWidget</class>
 <widget class="QWidget" name="QWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>x center</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>x range</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>x #points</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>y center</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>y #points</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>y range</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinXCenter">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
   <property name="value">
    <double>0.480000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinXRange">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
   <property name="value">
    <double>0.300000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinXPts">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>31</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinYRange">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
   <property name="value">
    <double>0.300000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinYPts">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>31</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinYCenter">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
   <property name="value">
    <double>0.730000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="button_auto_position">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>auto position</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>initial x</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>initial y</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinX0">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
   <property name="value">
    <double>0.480000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinY0">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
   <property name="value">
    <double>0.730000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>in initial scan window:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>new x:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>410</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>new y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>410</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>NiDaq X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>NiDaq Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>NiDaq AI:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineGalvoX">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineGalvoY">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePD">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinUpsample">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>upsample</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>directory</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textResultsDir">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;C:\Users\opticool\Documents\Mohamed\zmeter_25_04_01\position_data&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
